sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/MergeRowsExec.scala

```diff
@@ -0,0 +1,218 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.datasources.v2
+
+import org.roaringbitmap.longlong.Roaring64Bitmap
```

Comment:
Actually, this introduces a new direct dependency to `sql/core` module because this is the first `import` and the first use of `Roaring64Bitmap`. Is `Roaring64Bitmap` required instead of `RoaringBitmap`?

Reply:
Yes, `Roaring64Bitmap` is required in this case as values generated by `MonotonicallyIncreasingID` are 64 bit long values. I'll add a note.

Reply:
Added a note below.